arrow-annotations/build.gradle

```diff
@@ -9,3 +9,17 @@ apply from: "$DOC_CREATION"
 dependencies {
     api "io.kindedj:kindedj:$KINDEDJ_VERSION"
 }
+
+compileKotlin {
+    kotlinOptions {
+        useIR = true
+        freeCompilerArgs += ["-Xallow-jvm-ir-dependencies"]
```

Comment:
@danimontoya :raised_hands: according to the errors, it seems this arg should be provided for all the modules. To solve it, you can create a PR in `arrow` repository with a branch called `new-arrow-meta-integration` (the same name) and include it here: https://github.com/arrow-kt/arrow/blob/master/gradle/subproject.gradle You'll see `kotlinOptions` there. Then re-run these checks.